src/tabs/BTCTab/BitcoinPriceTab.jsx

```diff
@@ -0,0 +1,20 @@
+import React, { useState } from 'react';
+import MessariAPI from './MessariAPI';
+
+/*
+ * Displays BTC price and re-renders API when button is clicked
+ */
+const BitcoinPriceTab = () => {
+  const [count, setCount] = useState(0);
+  return(    
+    <>
+      <div>BTC Price USD: </div>
+      <MessariAPI count={ count }/>
+      <button onClick={() => setCount(count + 1)}>
+        Update BTC Price
+      </button>
+    </>
+  );
+}
+
+export default BitcoinPriceTab;
```

Comment:
make sure you add a newline here too

Reply:
I did. But i didn't set the vscode to always add an extra line when i save.  It messed up my input files.  I'll try to remember!